en/docs/develop/monitoring-capp-level-logs.md

```diff
@@ -1,26 +1,28 @@
 # Monitoring Composite Application-Level Logs
 
-The advantage of having per-composite application logs is that it is very easy to analyze/monitor what went wrong in this particular composite application (integration) by looking at the logs. 
+The advantage of having per-composite application logs is that it is straightforward to analyze/monitor what went wrong in a particular composite application (integration) by looking at the logs. 
 
-## Enabling log4j2 for capp logging
+## Enabling log4j2 for Composite Application logging
```

Comment:
Titles should be in sentence case. [1] https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-titles